rules/kube_apiserver.libsonnet

```diff
@@ -157,38 +157,50 @@
           }
           for verb in verbs
         ] + [
+          {
+            record: 'cluster_verb_scope:apiserver_request_duration_seconds_count:increase1h',
+            expr: |||
+              sum by (%(clusterLabel)s, verb, scope) (increase(apiserver_request_duration_seconds_count[1h]))
+            ||| % $._config,
+          },
+          {
+            record: 'cluster_verb_scope_le:apiserver_request_duration_seconds_bucket:increase1h',
+            expr: |||
+              sum by (%(clusterLabel)s, verb, scope, le) (increase(apiserver_request_duration_seconds_bucket[1h]))
+            ||| % $._config,
+          },
           {
             record: 'apiserver_request:availability%s' % SLODays,
             expr: |||
               1 - (
                 (
                   # write too slow
-                  sum by (%(clusterLabel)s) (increase(apiserver_request_duration_seconds_count{%(kubeApiserverWriteSelector)s}[%(SLODays)s]))
+                  sum by (%(clusterLabel)s) (avg_over_time(cluster_verb_scope:apiserver_request_duration_seconds_count:increase1h{%(kubeApiserverWriteSelector)s}[%(SLODays)s]) * 24 * %(days)s)
```

Comment:
We should be able to slightly optimize the evaluations by pre-computing these recording rule as in #537:
- `cluster_verb_scope:apiserver_request_duration_seconds_bucket:increase%s' % SLODays`
- `cluster_verb_scope_le:apiserver_request_duration_seconds_bucket:increase%s' % SLODays`

This shouldn't amount to much but considering the number of series, it might be worth it.